src/robusta/runner/config_loader.py

```diff
@@ -245,26 +245,24 @@ def __prepare_runtime_config(
         new_sinks = SinksRegistry.construct_new_sinks(runner_config.sinks_config, existing_sinks, registry)
         sinks_registry = SinksRegistry(new_sinks)
 
-        # TODO we will replace it with a more generic mechanism, as part of the triggers separation task
-        # First, we load the internal playbooks, then add the user activated playbooks
-        # Order matters. Internal playbooks, should be added first, and run first
-        active_playbooks = [
-            PlaybookDefinition(
-                triggers=[{"on_kubernetes_any_resource_all_changes": {}}],
-                actions=[{"cluster_discovery_updates": {}}],
-            )
-        ]
-        if runner_config.active_playbooks:
-            active_playbooks.extend(runner_config.active_playbooks)
+        active_playbooks = []
+        if runner_config.playbooks_map:
+            active_playbooks.extend(runner_config.playbooks_map.values())
         else:
             logging.warning("No active playbooks configured")
 
-        playbooks_registry = PlaybooksRegistryImpl(
-            active_playbooks,
-            actions_registry,
-            runner_config.global_config,
-            sinks_registry.default_sinks,
-        )
+        try:
```

Comment:
Is it better to just let the error be thrown here?
If we return undefined playbooks registry it will fail in many other places